posthog/queries/trends/test/test_formula.py

```diff
@@ -302,8 +302,10 @@ def test_event_properties(self):
 
     def test_compare(self):
         response = self._run({"date_from": "-1dStart", "compare": True})
+        self.assertEqual(response[0]["days"], ["2020-01-03", "2020-01-04"])
+        self.assertEqual(response[1]["days"], ["2020-01-01", "2020-01-02"])
         self.assertEqual(response[0]["data"], [1350.0, 0.0])
-        self.assertEqual(response[1]["data"], [1200.0, 1350.0])
+        self.assertEqual(response[1]["data"], [0.0, 1200.0])
```

Comment:
as above...

so the test was wrong because the date was picking the wrong ceiling/floor?

Could we put matches either side of the correct date boundary so you get a fail if that boundary changes? Or is that covered elsewhere?

Reply:
Specifically this is because the two date ranges overlapped because of https://github.com/PostHog/posthog/pull/12595/files/e5dbcbeab5fba53218740a8018d470b951e1efc0#diff-d5165bbe525763f97e122cadcabfbdd0538cd96e9447c766a1fe89a4116ac946L24. That timedelta is now only added in the narrow case where it's good for the UX (https://github.com/PostHog/posthog/pull/12595/files/e5dbcbeab5fba53218740a8018d470b951e1efc0#diff-83e1564f93aa8bbe220b6e93a740cd80e32f6ab81a13346a2a6310ea41e69a8bR498-R506), otherwise it isn't, so this is correct-er